packages/esm-messages-app/src/routes.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This module won't do anything with nothing in the routes file, but I see you talked about this earlier in the PR conversation, so ok